suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -267,7 +267,7 @@ tests:
       config:
         command: add
         id: client.1                      # client Id (<type>.<Id>)
-        node: node4                       # client node
+        node: node5                       # client node
```

Comment:
if you are planning to change the node name please update the node name in test desc as well.

Reply:
Done.